docs/release/role-handbooks/communications/README.md

```diff
@@ -0,0 +1,208 @@
+# Onboarding Notes
+
+Welcome to the Cluster API Comms Team onboarding documentation!
+
+## Overview
+
+- Understand Release Process:
+  - Get to know how project's release process works.
+  - Walk through the [release note generation process](#create-pr-for-release-notes) and try to generate notes by yourself. This is the most important process the comms team is in charge of.
+  - Familiarize yourself with the release notes tool [code](https://github.com/kubernetes-sigs/cluster-api/tree/main/hack/tools/release). You'll probably need to update this code during the release cycle to cover new cases or add new features.
+- Documentation familiarity:
+  - Explore project's documentation and start learning how to update and maintain it.
```

Comment:
I think it would help to list what docs to look at as well as a sample PR for updating it.